crossplane.yaml

```diff
@@ -40,7 +40,9 @@ metadata:
 
 spec:
   crossplane:
-    version: ">=v0.14.0-0"
+    version: ">=v1.0.0-0"
```

Comment:
this will work in dev, but we'll need to get Crossplane v1.0 into staging/prod before pushing this version to those environments

Reply:
yes, we should hold on this until we at least get a Crossplane v1.0 RC into prod

Reply:
found/fixed a minor issue: https://github.com/upbound/platform-ref-aws/pull/37/files#diff-78d366fd640fa3f1284c1d63bdb64e99893194f62742698bd17cceafca782843R158